### PR TITLE
T1

### PR DESCRIPTION
**Pull request by yvflerov**
some train edits

### CODE REVIEW

**Comment by Dshks** [comment]
comment one

File: f1 [hunk 1]
```diff
@@ -3,3 +3,4 @@ f1
 f1
 f1
 iis
+t1 commint2
```
Comment (Dshks) [inline f1]: comment string

**Comment by Dshks** [comment]
- [X] Write the code
- [ ] Write all the tests
- [ ] Document the code

**Comment by yvflerov** [comment]
- [X] Write the code
- [ ] Write all the tests
- [ ] Document the code

**Comment by Dshks** [comment]
:bug:

**Comment by Dshks** [comment]
@yvflerov

**Comment by Dshks** [comment]
- [ ] 

1. 

-

**Pull request merged**